README.md

```diff
@@ -95,6 +98,70 @@ export const config = {
 
 ```
 
+This script will not fetch configs from Eppo, only from Vercel Config Store.
+
+To fetch configs from Eppo and store them in Vercel Config Store, you need to create a Vercel Function.
+
+Example:
+
+`pages/api/eppo-prefetch.ts`
+
+```ts
+export const runtime = 'nodejs';
+
+import { IAssignmentLogger, prefetchConfig } from '@eppo/vercel-edge-sdk';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const assignmentLogger: IAssignmentLogger = {
+  logAssignment(assignment) {
+    console.log('assignement', assignment)
+  },
+};
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+
+  try {
+    if (!process.env.EDGE_CONFIG) {
+      throw new Error('Define EDGE_CONFIG env variable');
+    }
+
+    if (!process.env.EDGE_CONFIG_STORE_ID) {
+      throw new Error('Define EDGE_CONFIG_STORE_ID env variable')
+    }
+
+    if (!process.env.EDGE_CONFIG_TOKEN) {
+      throw new Error('Define EDGE_CONFIG_TOKEN env variable')
+    }
+
+    if (!process.env.INTERNAL_FEATURE_FLAG_API_KEY) {
+      throw new Error('Define INTERNAL_FEATURE_FLAG_API_KEY env variable')
+    }
+
+    prefetchConfig({
```

Comment:
should this be `await`ed?

Reply:
Well, it is better not.
We don't actually care about the response in the middleware, and Vercel Logs will show errors in function if there are some.
This way this gonna be faster and non-blocking